### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by sumit-vishwakarma**

### CODE REVIEW

File: html/adminPannel.html [hunk 1]
```diff
@@ -0,0 +1,678 @@
...
+            <button type="button" class="btn btn btn-outline-danger m-1" id="logoutButton">Logout</button>
+          </nav>
+          <div id="screenContent">
+            <!-- <div class="d-flex justify-content-between my-4">
+              <div>
+                <h3>Upcoming Meetings</h3>
+              </div>
+              <div>
+                <button type="button" class="btn btn-success" id="myBtn">
+                  Add Meeting
```
Comment (govind-sharma-4112) [inline html/adminPannel.html]: remove commented code

File: html/adminPannel.html [hunk 2]
```diff
@@ -0,0 +1,678 @@
...
+      </div>
+    </div>
+
+    <!-- Approve Room Modal -->
+    <div
+      class="modal fade"
+      id="approveRoomModal"
+      tabindex="-1"
+      role="dialog"
+      aria-labelledby="deleteRoomModalLabel"
```
Comment (govind-sharma-4112) [inline html/adminPannel.html]: remove unused code

File: index.html [hunk 3]
```diff
@@ -3,724 +3,52 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-    <title>Room Booking System</title>
-
-    <link rel="stylesheet" href="./css/style.css" />
+    <title>Document</title>
```
Comment (govind-sharma-4112) [inline index.html]: update the name

File: js/dashboard.js [hunk 4]
```diff
@@ -43,10 +62,11 @@ const dashboardScreen = (meetings) => {
                 </thead>
                 <tbody>
                     ${meetings?.map((item) => {
+                      let n = userNameForMeeting.filter(u=>u.emp_id == item.fk_emp_id);
```
Comment (govind-sharma-4112) [inline js/dashboard.js]: update the variable name

File: js/login.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        let loggedInAdminId = res.data.data[0];
+        console.log(loggedInAdminId)
+        if (loggedInAdminId.role == 1) {
+          localStorage.setItem("loggedInAdminId", loggedInAdminId.emp_id);
+          location.replace("./html/adminPannel.html");
+        } else {
+          alert("Acces Denied");
+        }
+        setTimeout(() => {
+          //   location.reload();
```
Comment (govind-sharma-4112) [inline js/login.js]: remove setTimeout

File: index.html (hunk 3 above)
Comment (sumit-vishwakarma) [reply index.html]: Done

**Pull request merged**